libs/execution_context/ebpf_program.c

```diff
@@ -25,6 +25,8 @@
 static size_t _ebpf_program_state_index = MAXUINT64;
 #define EBPF_MAX_HASH_SIZE 128
 
+#define CAPABILITY_CONTEXT_HEADER(capabilities) (capabilities & PROGRAM_DATA_CAPABILITY_CONTEXT_HEADER)
```

Comment:
This define should also go away and just use something like
capabilities.program_data_compatibility.context_header
which also supports intellisense.